webhook_to_fedora_messaging_messages/github/github.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-FileCopyrightText: 2024 Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+from ..base import Webhook2FedMsgBase
+from . import summarize_repository_event
+
+
+class GithubMessageV1(Webhook2FedMsgBase):
+    @property
+    def app_name(self):
+        return "Github"
+    
+    @property
+    def target_id(self):
+        return self.body['headers']['X-Github-Hook-Installation-Target-Id']
+    
+    @property
+    def signature(self):
+        """
+            SHA1 signature of the request.
+        """
+        return self.body['headers']['X-Hub-Signature']
+    
+    @property
+    def delivery(self):
+        """
+            A globally unique identifier (GUID) to identify the event.
+        """
+        return self.body['headers']['X-Github-Delivery']
+    
+    @property
+    def signature_sha256(self):
+        """
+            SHA1-256 signature of the request.
+        """
+        return self.body['headers']['X-Hub-Signature-256']
+    
+    @property
+    def event(self):
+        return self.body['headers']["X-Github-Event"]
+
+    @property
+    def event_type(self):
+        return self.body['headers']["X-Github-Hook-Installation-Target-Type"]
+
+    @property
+    def summary(self):
+        if self.event_type == "repository":
+            return summarize_repository_event()
+        else:
+            return "Message has no summary"
```

Comment:
You can return the superclass' value if you don't know how to handle it here, for example with `return super().summary`

Also, the summary is a one-line description of what happened, and your `summarize_*` functions return multiple lines. Think of it as the subject of an email.

Reply:
Do you want me to shorten the information in the summary and make it one line?